source/scripts/Background/migration/v5_4_2.ts

```diff
@@ -29,9 +31,9 @@ const VERSION = '5.4.2';
 
 const MigrateRunner = async (oldState: any) => {
   const DAG_ASSET_KEY = 'constellation';
-  const assetsFiltered = filterObjectByKey(oldState.assets, USDC_ASSET_KEY);
+  const assetsFiltered = filterObjectByKey(oldState.assets, [USDC_ASSET_KEY, PACA_ASSET_KEY]);
   const [assetsPart1, assetsPart2] = splitObjectByKey(assetsFiltered, DAG_ASSET_KEY);
-  const customAssetsFiltered = oldState.vault.customAssets.filter((asset: any) => asset.id !== USDC_ASSET_KEY);
+  const customAssetsFiltered = oldState.vault.customAssets.filter((asset: any) => ![USDC_ASSET_KEY, PACA_ASSET_KEY].includes(asset.id));
   try {
     const newState = {
       ...oldState,
```

Comment:
Can you explain this part please?

Reply:
This logic removes the USDC.dag and PACA tokens from local storage. It also removes both tokens from the `customAssets` array if, for some reason, the user added it manually.